webcompat/static/js/lib/wizard/ui-utils.js

```diff
@@ -17,11 +17,8 @@ export const scrollToElement = (element, delay = 250) => {
   setTimeout(() => {
     const topOfElement =
       window.pageYOffset + element.getBoundingClientRect().top - HEADER_HEIGHT;
-    window.scroll({ top: topOfElement, behavior: "smooth" });
+    $("html, body").animate({ scrollTop: topOfElement });
```

Comment:
Added it to make scrolling smooth in IE. So it makes it smooth in Safari as well :)